src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs

```diff
@@ -103,8 +122,7 @@ public async Task<ConnectorType> ResolveTableAsync(string tableName, Cancellatio
             return connectorType;
         }
 
-        internal static bool IsSql(string uriPrefix) => uriPrefix.Contains("/sql/");
-
+        [Obsolete]
         internal static bool UseV2(string uriPrefix) => uriPrefix.Contains("/sql/") ||
                                                         uriPrefix.Contains("/zendesk/");
```

Comment:
Since this is ultimately used to determine the endpoint, can we remove from here and push higher?

Reply:
I'm not sure where you'd put it
This is used twice in this class, twice in CdpDataSource and once in CdpTable, always in [Obsolete] methods

If we want to absorb the breaking change fully we can jsut delete all these [Obsolete] things
Just let know if is you agree/want that?